<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenghitungAtmosferISA</class>
 <widget class="QDialog" name="MenghitungAtmosferISA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #2776ea;</string>
  </property>
  <widget class="QPushButton" name="hitung">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Hitung</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tinggiTerbangHasil">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;
color: white;</string>
   </property>
   <property name="text">
    <string>Tinggi Terbang        : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;
color: white;</string>
   </property>
   <property name="text">
    <string>Temperatur Udara :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturUdara">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="tekananUdara">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;
color: white;</string>
   </property>
   <property name="text">
    <string>Tekanan Udara        :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;
color: white;</string>
   </property>
   <property name="text">
    <string>Kepadatan Udara   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kepadatanUdara">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: black;</string>
   </property>
   <property name="text">
    <string>Lapisan  Udara        :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namaLapisan">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>326</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: white;
background: black;</string>
      </property>
      <property name="text">
       <string>Masukan Tinggi Terbang : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tinggiTerbang">
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
